admin/src/main/resources/io/buoyant/admin/js/src/router_client.js

```diff
@@ -169,8 +170,14 @@ define([
 
       function toggleClientDisplay(expand) {
         if (expand) {
+          $contentContainer.css({"border": colorBorder});
```

Comment:
The tiniest of nitpicky bike sheds: I think in the mocks the corners are rounded.